web/src/views/LandingView.vue

```
@@ -0,0 +1,61 @@
+<script setup lang="ts">
+import { MessageSquarePlus, Vote, Map } from 'lucide-vue-next'
```

Biome (2.3.14)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)